src/Ch9/Ch9.Shared/MainPage.xaml.cs

```diff
@@ -52,5 +54,14 @@ private void OnItemsSourceChanged(DependencyObject sender, DependencyProperty dp
 				listView.SelectedIndex = 0;
 			}
 		}
+
+		private void VisualStateGroupCurrentStateChanged(object sender, VisualStateChangedEventArgs e)
+		{
+			//Allow screen rotation for this particular state
```

Comment:
Not sure what you mean

Reply:
If narrow and selected then allow screen rotation as we set preferences to none.